src/index.html

```diff
@@ -5,8 +5,482 @@
+          alt="Logo Moyo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#Apple" class="header__item-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Samsung" class="header__item-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Smartphones" class="header__item-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              class="header__item-link"
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Gadgets" class="header__item-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Tablets" class="header__item-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Photo" class="header__item-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Video" class="header__item-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__cards-first_row">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+
+          <p class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars card__rating-stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+
+            <p class="card__rating-reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+            <p class="card__price-sum">
+              $2,199
+            </p>
+          </div>
+
+          <div class="card__button">
+            <a
+              href="#"
+              data-qa="card-hover"
+              class="card__button-link"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+        <div class="card">
```

Comment:
```suggestion
        </div>
        
        <div class="card">
```
Don't forget to add empty lines between blocks of code